### PR TITLE
update to spfx 1.16.1

### PR DESCRIPTION
**Pull request by alexgomezlf**
update to Node 16
updtae from tslint to eslint
remove axios
remove fontawesome, replace with material icons

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require('@rushstack/eslint-config/patch/modern-module-resolution');
+module.exports = {
+  extends: ['@microsoft/eslint-config-spfx/lib/profiles/react'],
```
Comment (Ke-Jin-Laserfiche) [inline .eslintrc.js:3]: why is it from '@microsoft/eslint-config-spfx/lib/profiles/react'?

File: src/extensions/savetoLaserfiche/SavetoLaserficheCommandSet.ts [hunk 2]
```diff
@@ -33,241 +29,263 @@ export interface ISendToLfCommandSetProperties {
...
+  //checking whether the Sign-in Page configured or not
+  public async pageConfigurationCheck(): Promise<any> {
+    webpartconfigurations = '';
+    try {
+      const res = await fetch(
+        this.context.pageContext.web.absoluteUrl + `/_api/web/lists/getbytitle('Site Pages')/items`,
+        {
+          method: 'GET',
+          headers: {
+            Accept: 'application/json',
```
Comment (Ke-Jin-Laserfiche) [inline src/extensions/savetoLaserfiche/SavetoLaserficheCommandSet.ts:111]: why is 'Accept' not in quote?

File: src/webparts/sendToLaserficheLoginComponent/components/SendToLaserficheLoginComponent.tsx [hunk 3]
```diff
@@ -1,14 +1,10 @@
...
-import styles from './SendToLaserficheLoginComponent.module.scss';
 import { ISendToLaserficheLoginComponentProps } from './ISendToLaserficheLoginComponentProps';
 import { ISendToLaserficheLoginComponentState } from './ISendToLaserficheLoginComponentState';
 import { SPComponentLoader } from '@microsoft/sp-loader';
 import CustomDailog from './SendToLaserficheCustomDialog';
 import { Navigation } from 'spfx-navigation';
-import { result } from 'lodash';
-import { CreateEntryResult, IRepositoryApiClient, RepositoryApiClient, PostEntryWithEdocMetadataRequest, PutFieldValsRequest, FileParameter,FieldToUpdate,ValueToUpdate } from '@laserfiche/lf-repository-api-client';
-import { LoginState } from '@laserfiche/types-lf-ui-components';
+import { CreateEntryResult, PostEntryWithEdocMetadataRequest, PutFieldValsRequest, FileParameter,FieldToUpdate,ValueToUpdate } from '@laserfiche/lf-repository-api-client';import { LoginState } from '@laserfiche/types-lf-ui-components';
```
Comment (Ke-Jin-Laserfiche) [inline src/webparts/sendToLaserficheLoginComponent/components/SendToLaserficheLoginComponent.tsx]: import in new line

File: config/sass.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/core-build/sass.schema.json"
```
Comment (Ke-Jin-Laserfiche) [inline config/sass.json:2]: what is this file used for?

File: config/serve.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/core-build/serve.schema.json",
+  "$schema": "https://developer.microsoft.com/json-schemas/spfx-build/spfx-serve.schema.json",
```
Comment (Ke-Jin-Laserfiche) [inline config/serve.json:2]: what is this schema file?

File: src/extensions/savetoLaserfiche/SavetoLaserficheCommandSet.ts (hunk 2 above)
Comment (alexgomezlf) [reply src/extensions/savetoLaserfiche/SavetoLaserficheCommandSet.ts:111]: single words in JSON don't need to be in quotes (like method, headers), but there will be another pass for linting and formatting

File: .eslintrc.js (hunk 1 above)
Comment (alexgomezlf) [reply .eslintrc.js:3]: I used this which was linked from the SpFx documentation to determine what to update: https://pnp.github.io/cli-microsoft365/cmd/spfx/project/project-upgrade/

File: config/sass.json (hunk 4 above)
Comment (alexgomezlf) [reply config/sass.json:2]: I believe this is used to compile the sass files

I used this which was linked from the SpFx documentation to determine what to update: https://pnp.github.io/cli-microsoft365/cmd/spfx/project/project-upgrade/

File: config/serve.json (hunk 5 above)
Comment (alexgomezlf) [reply config/serve.json:2]: I used this which was linked from the SpFx documentation to determine what to update: https://pnp.github.io/cli-microsoft365/cmd/spfx/project/project-upgrade/

File: src/webparts/sendToLaserficheLoginComponent/components/SendToLaserficheLoginComponent.tsx (hunk 3 above)
Comment (alexgomezlf) [reply src/webparts/sendToLaserficheLoginComponent/components/SendToLaserficheLoginComponent.tsx]: fixed

**Review by Ke-Jin-Laserfiche — APPROVED** [review]

**Pull request merged**